tests/node/rpc/messages/UnsubscribeVehicleDataResponseTests.js

```diff
@@ -1,8 +1,14 @@
 const SDL = require('../../../config.js').node;
+
+// messages
 const UnsubscribeVehicleDataResponse = SDL.rpc.messages.UnsubscribeVehicleDataResponse;
+
+// enums
 const FunctionID = SDL.rpc.enums.FunctionID;
 const MessageType = SDL.rpc.enums.MessageType;
 const VehicleDataType = SDL.rpc.enums.VehicleDataType;
+
+// structs
 const VehicleDataResult = SDL.rpc.structs.VehicleDataResult;
 
 const BaseRpcTests = require('./BaseRpcTests');
```

Comment:
**Optional:** On line 24 you could change `this.vehicleDataResult` to `this.handsOffSteering`.

Reply:
Based on previous experience, it is better to avoid changes not related to PR. Will add if PMs request this in the review.